persistence-model/src/main/resources/application.conf

```diff
@@ -77,6 +77,10 @@ cassandra-journal {
   connect-retries = 10
   connect-retry-delay = 5s
   reconnect-max-delay = 60s
+  authentication.username = ""
```

Comment:
The journal and snapshot implementations take default configuration from [here](https://github.com/akka/akka-persistence-cassandra/blob/master/core/src/main/resources/reference.conf). The `Akka` code reads these properties and uses them, all I've done here is override them. This is how we have been able to use the host, port, etc. all along. The `CassandraRepository` implementation uses the `Java` driver directly and that is why the implementation is different.